docs/mine/lotus/split-markets-miners.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Lotus Miner: Splitting main miner and markets service processes'
+description: ''
+breadcrumb: 'Splitting main miner and markets service processes'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+## Background
+
+Lotus v1.11.0 introduced the notion of `subsystems` in the `lotus-miner` process. Currently there are 4 subsystems, that can be enabled or disabled in the `config.toml` file:
+
+```
+[Subsystems]
+#  EnableMining = true
+#  EnableSealing = true
+#  EnableSectorStorage = true
+#  EnableStorageMarket = true
+```
+
+All these options are set to `true` by default. Until now, the monolith `lotus-miner` process has been responsible for all functionality.
+
+At the moment these `subsystems` are designed to be grouped into two distinct types of `lotus-miner` nodes:
```

Comment:
What happens if the user passes an unsupported configuration? Could we explain it in the doc?